authors/apps/authentication/urls.py

```diff
@@ -1,11 +1,11 @@
-from django.conf.urls import url
+from django.urls import path
 
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
```

Comment:
As a good practice, you can add app_name variable which indicates the name of the app

Reply:
This has been duly implemented